doc/api/stream.md

```diff
@@ -1754,8 +1754,9 @@ methods only.
 
 The `writable._writev()` method may be implemented in addition to
 `writable._write()` in stream implementations that are capable of processing
-multiple chunks of data at once. If implemented, the method will be called with
-all chunks of data currently buffered in the write queue.
+multiple chunks of data at once. If implemented, the first chunk will be written with 
+`_write()`. When the initial write is complete, buffered chunks in the write queue, if available, 
```

Comment:
I'm pretty sure our linter will complain that thisline is longer than 80 chars and has trailing white space. At least I *hope* it will flag those things. Run `make lint-md` (or `vcbuild lint-md` if on Windows) to run the documentation linter locally.

Reply:
Also: Welcome @tyof45 and thanks for the pull request!